CHANGES_NEXT_RELEASE

```diff
@@ -0,0 +1 @@
+- Fix: DEFAULT_ATTRIBUTE_TYPE is changed to Text (#1569)
```

Comment:
```suggestion
```

The change is the functional test part, not involving actual functional code in the library. Thus, the CHANGES_NEXT_RELEASE entry is not needed.